modules/benchmark/R/load_data.R

```diff
@@ -58,6 +58,16 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
   
   vars_used <- format$vars[vars.used.index, ]
   
+  # check wide format and transform to long
+  if(any(duplicated(vars_used$bety_name))){
```

Comment:
Hmm good point.. Can we trust that there will be a consistent naming pattern? I'll revise the wide2long transformation